tests/data/spe1_st/tests/test_webviz_ert.py

```diff
@@ -37,7 +26,7 @@ def _verify_keys_in_menu(dash_duo_handle, plugin, keys, selector):
 def test_webviz_parameter_comparison(get_ensemble_id, dash_duo):
     # here we need the poke storage first - to get this test running
     _ = get_ensemble_id
-    plugin = _gen_plugin(dash_duo, ParameterComparison)
+    plugin = setup_plugin(dash_duo, ParameterComparison)
```

Comment:
I think you are missing the `name` argument and also it might be a good idea to set the window size to what it was before the change `windowsize = (630, 2000)`

Reply:
ah thanks, of course! fixed. gonna run tests, and repush

Reply:
that is so funny - the tests seem to work out just fine, but jenkins actually catches the missing name argument O_o

Reply:
fixed!